go.mod

```diff
@@ -3,7 +3,6 @@ module github.com/k8ssandra/medusa-operator
 go 1.13
```

Comment:
Not necessarily this PR, but should we look at updating these builders to use 1.15?

Reply:
Absolutely yes. Let's do it in follow up PRs for both medusa-operator and for reaper-operator. I will create the issues.

Reply:
@burmanm https://github.com/k8ssandra/medusa-operator/issues/10